pages/search.js

```diff
@@ -112,7 +112,7 @@ export default function Search({ users }) {
         {!threeOrMore && (
           <Alert
             type="info"
-            message="You have to enter at least 3 characters to search for a user."
+            message="You have to enter at least 3 characters to search for a users,tags or languages."
```

Comment:
```suggestion
            message="You have to enter at least 3 characters to search for users,tags or languages."
```

Reply:
Thanks @swagataroy30 for pointing out the mistake it has been updated